bin/rnws.js

```diff
@@ -105,18 +105,18 @@ commonOptions(program.command('bundle'))
     });
     const targetPath = path.resolve(opts.bundlePath);
 
-    server.start();
-
-    fetch(bundleUrl).then(function(content) {
-      fs.writeFileSync(targetPath, content);
-      server.stop();
+    server.start().then(function() {
+      fetch(bundleUrl).then(function(content) {
```

Comment:
Can you chain the promises to avoid nesting? i.e.
```js
server.start().then(function() {
  return fetch(bundleUrl);
}).then(function(content) {
  ...
})
```